### PR TITLE
feat: change to new api

### PR DESCRIPTION
**Pull request by dungkm101digital**
### Why

Change ticket system to use new api

Link to Jira ticket:
https://acsdigital.atlassian.net/browse/APB-712
https://acsdigital.atlassian.net/browse/APB-710
https://acsdigital.atlassian.net/browse/APB-883

### CODE REVIEW

File: src/components/adb-create-ticket-details-component/index.tsx [hunk 1]
```diff
@@ -137,45 +142,46 @@ const ADBCreateTicketDetailsComponent: React.FC<ICreateTicketDetails> = ({
...
-                    )
+                    attachments.map((n) => {
+                      let filename = n.uri;
+                      if (Platform.OS === "ios") {
+                        filename = n.uri.replace("file:", "");
+                      }
+                      return RNFetchBlob.fs
+                        .readFile(decodeURIComponent(filename), "base64")
+                        .catch((r: any) => {
+                          console.log("readfFile catch: ", r);
```
Comment (sk101dio) [inline src/components/adb-create-ticket-details-component/index.tsx:167]: remove this log.

**Review by sk101dio — CHANGES_REQUESTED** [review]
just remove that log, rest looks good.

**Review by sk101dio — APPROVED** [review]
looks good

File: src/common/constants.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export enum TicketStatusValue {
   SUBMITTED = "Submitted",
-  IN_PROGRESS = "In-Progress",
+  IN_PROGRESS = "In Progress",
```
Comment (sharmal33) [inline src/common/constants.ts:3]: this change will brake the existing flow which was implemented by Minh

File: src/components/adb-create-ticket-details-component/index.tsx [hunk 3]
```diff
@@ -137,45 +142,42 @@ const ADBCreateTicketDetailsComponent: React.FC<ICreateTicketDetails> = ({
...
-              SuppliedName: "Minh",
               Description: values.complaint,
               Origin: "App",
+              accountId: account!.accountId,
+              contactId: account!.contactId,
             };
             try {
               const response = await ticketingService.createCase(payload);
-              if (response.success) {
+              if (response.serviceStatus === "SUCCESS") {
```
Comment (sharmal33) [inline src/components/adb-create-ticket-details-component/index.tsx:155]: plz get these("SUCCESS") values from utils file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/common/constants.ts (hunk 2 above)
Comment (dungkm101digital) [reply src/common/constants.ts:3]: Yes, this is the new api value.

File: src/components/adb-create-ticket-details-component/index.tsx (hunk 3 above)
Comment (dungkm101digital) [reply src/components/adb-create-ticket-details-component/index.tsx]: I've added this function to utils file 
```js 
export function checkIfServiceStatusSuccess(response: CreateCaseResponse) {
    return response.serviceStatus === 'SUCCESS';
}
```

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**